packages/calling/src/CallingClient/line/index.ts

```diff
@@ -70,7 +68,6 @@ export default class Line extends Eventing<LineEventTypes> implements ILine {
     mutex: Mutex,
     primaryMobiusUris: string[],
     backupMobiusUris: string[],
-    callingClientEmitter: (event: EVENT_KEYS, clientError?: CallingClientError) => void,
     logLevel: LOGGER,
```

Comment:
Apologies for noticing this now, the below method in this file looks like a dead code with no references. 
`keepaliveTimer` is  started automatically in register.ts file once a registration is successful and developers need not manually send/refresh the timer manually.
Earlier this was a public method in CallingClient but was never invoked by any client. 

@Kesari3008  Shall we remove this? Lets remove any declarations in any interfaces as well (line and CallingClient)

`134 Line:   private async sendKeepAlive(deviceInfo: IDeviceInfo): Promise<void> {`

Reply:
@BhargavSatya  Thanks for bringing this to notice, I missed this actually. @meta-dipanshu-sharma  You can remove this method completely, Bhargav has renamed the method sendKeepalive to startKeepaliveTimer and it exists in the register.ts file. It's a private function so we can keep it private as it is only going to be invoked internally.

Reply:
Removed `sendKeepAlive` wherever used.